azure_functions/reporting/run_report/__init__.py

```
@@ -0,0 +1,240 @@
+    except ValueError:
+        return func.HttpResponse("Invalid JSON", status_code=400)
+
+    run_id = body.get("run_id")
+    if not run_id:
+        return func.HttpResponse("run_id is required", status_code=400)
+
+    credential = _get_credential()
+
+    try:
+        events = _query_events_with_fallback(run_id, credential)
+    except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.6)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)